src/bitmessagecli.py

```diff
@@ -521,90 +541,98 @@ def listAdd(): #Lists all of the addresses and their info
     print '\n     --------------------------------------------------------------------------'
     print '     | # |       Label       |               Address               |S#|Enabled|'
     print '     |---|-------------------|-------------------------------------|--|-------|'
-    for addNum in range (0, numAddresses): #processes all of the addresses and lists them out
-        label   =    (jsonAddresses['addresses'][addNum]['label'  ]).encode('utf')              # may still misdiplay in some consoles
+    for addNum in range(0, numAddresses):  # processes all of the addresses and lists them out
+        label = (jsonAddresses['addresses'][addNum]['label']).encode(
+            'utf')              # may still misdiplay in some consoles
```

Comment:
utf-8?

Reply:
Not a style change but nice catch. It deserves its own ticket: #1240.